src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,93 @@
+import clsx from 'clsx';
+import React from 'react';
+import Icon from '../Icon';
+import styles from './Avatar.module.css';
+
+export interface Props {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+  /**
+   * The user name associated with this avatar
+   */
+  name?: string;
```

Comment:
Maybe specify we want in "Firstname Lastname"?

Reply:
@jinlee93 I'll bring this up in discussion, but `name` could be anything. Right now it assumes fname lname with some parsing to exclude middle names, but I think the ideal state is either some object representing user data (name, login, email, etc.) or renaming this such that it does not imply their "name" name. Will figure all that out today :)

Reply:
That does sound more robust, thanks!